ui/apps/platform/cypress/integration/accessControl/accessControlOrigin.test.js

```diff
@@ -0,0 +1,174 @@
+        const staticResponseMap = {
+            [permissionSetsAlias]: {
+                fixture: 'auth/declarativePermissionSet.json',
+            },
+        };
+        visitAccessControlEntities(permissionSetsKey, staticResponseMap);
+
+        cy.get('td[data-label="Origin"]').should('have.text', 'Declarative');
+        cy.get('td.pf-c-table__action button[aria-label="Actions"]').should('be.disabled');
+    });
+
+    it('list link for declarative permission set goes to form which has label instead of button and disabled input values', () => {
+        const staticResponseMap = {
+            [permissionSetsAlias]: {
+                fixture: 'auth/declarativePermissionSet.json',
+            },
+        };
+        visitAccessControlEntities(permissionSetsKey, staticResponseMap);
+
+        const entityName = 'permission-set-test-name';
+        clickEntityNameInTable(permissionSetsKey, entityName);
+
+        cy.get(`h2:contains("${entityName}")`);
+        cy.get(`li.pf-c-breadcrumb__item:nth-child(2):contains("${entityName}")`);
+
+        cy.get('.pf-c-label:contains("Declarative")').should('exist');
+
+        cy.get(selectors.form.notEditableLabel).should('exist');
+        cy.get(selectors.form.editButton).should('not.exist');
+
+        cy.get(selectors.form.inputName).should('be.disabled');
+        cy.get(selectors.form.inputDescription).should('be.disabled');
+    });
+
+    it('declarative roles in list view have correct origin and are disabled', () => {
+        const staticResponseMap = {
+            [rolesAlias]: {
+                fixture: 'auth/declarativeRole.json',
+            },
+        };
+        visitAccessControlEntities(rolesKey, staticResponseMap);
+
+        cy.get('td[data-label="Origin"]').should('have.text', 'Declarative');
+        cy.get('td.pf-c-table__action button[aria-label="Actions"]').should('be.disabled');
+    });
+
+    it('list link for declarative role goes to form which has label instead of button and disabled input values', () => {
+        const staticResponseMap = {
+            [rolesAlias]: {
+                fixture: 'auth/declarativeRole.json',
+            },
+        };
+        visitAccessControlEntities(rolesKey, staticResponseMap);
+
+        const entityName = 'test-role-name';
+        clickEntityNameInTable(rolesKey, entityName);
+
+        cy.get(`h2:contains("${entityName}")`);
+        cy.get(`li.pf-c-breadcrumb__item:nth-child(2):contains("${entityName}")`);
+
+        cy.get('.pf-c-label:contains("Declarative")').should('exist');
+
+        cy.get(selectors.form.notEditableLabel).should('exist');
+        cy.get(selectors.form.editButton).should('not.exist');
+
+        cy.get(selectors.form.inputName).should('be.disabled');
+        cy.get(selectors.form.inputDescription).should('be.disabled');
+    });
+
+    it('declarative auth providers in list view have correct origin and are disabled', () => {
```

Comment:
What do you think about either of the following principles to order pairs of tests in this file?

Either way, auth provider at the end seems out of order. To make clear for reader and future writer if there is need to add any more tests, how about a comment preceding each pair to make the organization clear?

1. Alphabetical order
    * access scope
    * auth provider
    * permission set
    * role
3. UI tab order
    * Auth providers
    * Roles
    * Permission sets
    * Access scopes

Reply:
I chose alphabetical as it matches order in which test files are displayed in `integration/accessControl` folder in IDE:
* `accessControlAccessScopes.test.js`
* `accessControlAuthProviders.test.js`
* `accessControlPermissionSets.test.js`
* `accessControlRoles.test.js`